resources/function_help/json/z_max

```diff
@@ -2,9 +2,12 @@
   "name": "z_max",
   "type": "function",
   "groups": ["GeometryGroup"],
-  "description": "Returns the maximum z coordinate of a geometry.",
+  "description": "Returns the maximum z coordinate of a geometry, or Null if the geometry has no z value.",
```

Comment:
Not related to this PR specifically, but we probably should do a scan and unify how "null" is represented in these help files. There's a mix of "NULL"/"Null"/"null"